pyoptsparse/pyALPSO/pyALPSO.py

```diff
@@ -95,7 +95,9 @@ def __init__(self, *args, **kwargs):
             "parallelType": [str, ""],  # Type of parallelization ('' or 'EXT')
         }
         self.informs = {}
-        Optimizer.__init__(self, "ALPSO", category, self.defOpts, self.informs, *args, **kwargs)
+        super().__init__(
+            self, "ALPSO", category=category, defOptions=self.defOpts, informs=self.informs, options=options
```

Comment:
Random: are we actually using the `category` attribute anywhere?

Reply:
Nope. This was designed to be kind of an attribute that gets queried I think, but it is not currently used. Nobody will instantiate "SNOPT", then query "SNOPT.category" and be surprised to find out that it is a local optimizer... but I'm not inclined to remove it since it might be useful. Let me know if you think otherwise.

Reply:
Plus, `baseclasses` now require a category defined so this fits nicely with that.

Reply:
Yeah, I was wandering if we were actually using that attribute in `baseclasses` as well. I agree we should not remove this.